lib/3scale_toolbox/commands/product_command/copy_command.rb

```diff
@@ -0,0 +1,78 @@
+require '3scale_toolbox/commands/product_command/copy_command/delete_target_backend_usages_task'
+require '3scale_toolbox/commands/product_command/copy_command/copy_backends_task'
+
+module ThreeScaleToolbox
+  module Commands
+    module ProductCommand
+      class CopySubcommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+
+        def self.command
+          Cri::Command.define do
+            name        'copy'
+            usage       'copy [opts] -s <source_remote> -d <target_remote> <source_product>'
```

Comment:
Can we change dashes instead of underscores for keywords with multiple words as we try to do in other places?

Reply:
:+1: